var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -23,6 +24,10 @@
 #    format returned by platform.system() and 'arch' by platform.machine()
 
 _versions = {
+    '11.4.0': {
```

Comment:
Thanks!

Any supported compiler changes that we should put in `CudaPackage`?

Reply:
Not for `x86_64` and `ppc64le`.

However I see no entries in `lib/spack/spack/build_systems/cuda.py` about `aarch64`. I'll consult colleagues before open an additional PR for those (and I will do some testing as well eventually).

Reply:
I am going to kick-off a conversation internally about this point, we all been a bit distracted by many virtual conferences in the past 3 weeks. I'd suggest to progress with the merge now while we make a proper assessment.

Reply:
@fspiga no worries, ping when this is ready for merge :)